data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/lib/flow-lib.xqy

```diff
@@ -369,7 +383,12 @@ declare function flow:run-collector(
   $options as map:map) as xs:string*
 {
   let $module-name := hul:get-module-name($module-uri)
-  let $ns := $PLUGIN-NS || fn:lower-case($module-name)
+  let $filename as xs:string := hul:get-file-from-uri($module-uri)
+  let $type := flow:get-type($filename)
+  let $ns :=
+    if ($type eq "sjs") then ()
```

Comment:
flow:get-type() actually uses the registered mime types. I have it returning "sjs" which, to your point, should be an enum. It should probably be:

```xquery
declare variable $TYPE-JAVASCRIPT := "type-javascript";
```

I'll make the change. Thx.